tests/test_core.py

```diff
@@ -255,3 +255,9 @@ def test_get_subset():
 
     assert set(x.keys()) == set(['CESM', 'FOO'])
     assert list(x.values()) == [2, 2]
+
+    x = _get_subset(df, C=['control', 'hist'], D=['NO2'], force_all_on='B')
```

Comment:
Would `require_all_on` be a better name than `force_all_on`? I think it would.

Reply:
Done!